src/libcommon/utility/types.h

```diff
@@ -364,10 +366,21 @@ struct ItemType {
 
 enum class AppStateKey {
     // Adding a new key here requires to add it in insertDefaultAppState in parmsdbappstate.cpp
-    LastServerSelfRestart,
-    LastClientSelfRestart,
-
-    Unknown,  //!\ keep in last position (For tests) /!\\ Used for initialization, will throw error if used.
+    LastServerSelfRestartDate,
+    LastClientSelfRestartDate,
+    LastSuccessfulLogUploadeDate,
```

Comment:
Typo `Uploade`

Reply:
Fix in commit https://github.com/Infomaniak/desktop-kDrive/pull/117/commits/f2bb135b7e2533638e6c36483f7c1d50ef921b53